doc/source/user_guide/groupby.rst

```diff
@@ -641,6 +641,9 @@ accepts the special syntax in :meth:`GroupBy.agg`, known as "named aggregation",
   to make it clearer what the arguments are. As usual, the aggregation can
   be a callable or a string alias.
 
+.. versionchanged:: 1.4.0
+   ``pandas.NamedAgg`` is no longer a ``collections.namedtuple`` but now a ``typing.NamedTuple``.
```

Comment:
oh this is fine

Reply:
is this relevant? the mro is the same? `typing.NamedTuple` and `collections.namedtuple` are just factory functions for tuple subclasses? Is there any user facing change that is relevant?

Reply:
Okay, that makes sense. I will revert these doc changes.